test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Link.html

```diff
@@ -31,14 +31,14 @@
     
 
     <!-- Globbed link tag with existing file and static href -->
-    <link href="/site.css" rel="stylesheet" /><link href="/sub/site2.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/site.css" /><link rel="stylesheet" href="/sub/site2.css" />
 
     <!-- Globbed link tag with existing file and static href should dedupe -->
-    <link href="/site.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/site.css" />
 
     <!-- Fallback to static href -->
-    <link href="/site.min.css" rel="stylesheet" data-extra="test" />
-<meta name="x-stylesheet-fallback-test" class="hidden" /><script>!function(a,b,c){var d,e=document,f=e.getElementsByTagName("SCRIPT"),g=f[f.length-1].previousElementSibling,h=e.defaultView&&e.defaultView.getComputedStyle?e.defaultView.getComputedStyle(g):g.currentStyle;if(h&&h[a]!==b)for(d=0;d<c.length;d++)e.write('<link rel="stylesheet" href="'+c[d]+'"/>')}("visibility","hidden",["\/site.css"]);</script>
+    <link rel="stylesheet" data-extra="test" title="&quot;the&quot; title" href="/site.min.css?a=b&amp;c=d" />
```

Comment:
don't want `&amp;` here.  see TODO item in PR description.  note this encoding is likely not a problem when using static files for CSS and script resources.